src/main/java/org/jenkinsci/plugins/workflow/libs/ResourceStep.java

```diff
@@ -49,6 +51,15 @@
     public String getResource() {
         return resource;
     }
+    
+    public String getLibraryName() {
+        return libraryName;
+    }
+    
+    @DataBoundSetter
+    public void setLibraryName(String libraryName) {
+        this.libraryName = libraryName;
```

Comment:
I would use `= Util.fixEmptyAndTrim(libraryName)` to remove surrounding whitespace from `libraryName`.